Description: Patch the sage startup script
 This patch removes a check for if we are in a sage source directory.
Origin: https://github.com/cschwan/sage-on-gentoo/blob/master/sci-mathematics/sage/files/sage-7.4-exec.patch
Forwarded: not-needed

--- a/sage/src/bin/sage
+++ b/sage/src/bin/sage
@@ -217,26 +217,6 @@
 
 # Prepare for running Sage, either interactively or non-interactively.
 sage_setup() {
-    # Check that we're not in a source tarball which hasn't been built yet (#13561).
-    if [ ! -z "$SAGE_LOCAL" ] && [ ! -x "$SAGE_LOCAL/bin/sage" ]; then
-        echo >&2 '************************************************************************'
-        echo >&2 'It seems that you are attempting to run Sage from an unpacked source'
-        echo >&2 'tarball, but you have not compiled it yet (or maybe the build has not'
-        echo >&2 'finished). You should run `make` in the Sage root directory first.'
-        echo >&2 'If you did not intend to build Sage from source, you should download'
-        echo >&2 'a binary tarball instead. Read README.txt for more information.'
-        echo >&2 '************************************************************************'
-        exit 1
-    fi
-
-    maybe_sage_location
-
-    if [ ! -d "$IPYTHONDIR" ]; then
-        # make sure that $DOT_SAGE exists so that ipython will happily
-        # create its config directories there.  If DOT_SAGE doesn't
-        # exist, ipython complains.
-        mkdir -p -m 700 "$DOT_SAGE"
-    fi
     sage-cleaner &>/dev/null &
 }
 
